infrastructure/bin/infrastructure.ts

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env node
-import 'source-map-support/register';
 import { App } from 'aws-cdk-lib';
-import { DynamoStack } from '../lib/dynamo-stack';
+import 'source-map-support/register';
 import { CognitoStack } from '../lib/cognito-stack';
+import { DynamoStack } from '../lib/dynamo-stack';
 
 const app = new App();
 
-new CognitoStack(app, 'sweep-cognito-stack');
-new DynamoStack(app, 'sweep-dynamo-stack');
+const dynamoStack = new DynamoStack(app, 'sweep-dynamo-stack');
+new CognitoStack(app, 'sweep-cognito-stack', dynamoStack.usersTable);
```

Comment:
I don't love directly passing the Users table in here. It makes it easy to add the correct permissions, but it might be better to handle that ourselves anyways. I think we should have the CDK output the ARN of the Users table for the CognitoStack to use.